script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -1126,7 +1126,7 @@ Explain yourself!
 -- Page 414, Offset 17606. Glued. Mods: @x.
```

Comment:
I actually would like to propose rewording the previous line to 'What I want to say is:' because that makes more sense in the context of an internal dialog with himself as opposed to a third party.

Reply:
This isn't internal dialogue though, he's saying this out loud. "Here's what I want to say" feels more natural to me as an expression in that context.

Reply:
even though its not internal dialogue I still kinda agree with Kurisu's change, but not that strongly. I'm ok leaving it as is

Reply:
I think my hesitation with the change is the ending on an `is` before the pause of the colon. If I say it out loud, it feels more like a sentence that has been cut off than one that has come to a declarative pause because the inflection of the `is` doesn't have that finality.  `What I want to say is this:` makes more sense for that formulation, to me at least.

Reply:
I still don't really agree, but I also don't have any real problem with the old one so I don't think this should block any longer.

ship it